packages/@vue/cli-plugin-e2e-cypress/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "@vue/cli-shared-utils": "^4.2.2",
-    "cypress": "^3.8.3",
+    "cypress": "^4.0.1",
```

Comment:
FWIW the latest version is now `4.3.0` :)

Reply:
Can we use a  Cypress >=4.5.0 as there is a dependency down the chain on cypress on mocha 2.4.5 that is reeeay old and causes issues with other plugins? Many thanks!

And any luck for this pr to be merged soon? :)

Reply:
why not use 4.4.0+ to support TS?